mindjunkies/forums/views.py

```
@@ -177,3 +178,65 @@ def post(self, request, *args, **kwargs):
             
         }
         return render(request, "forums/reply_form.html", context)
+    
+class LikeToggleView(LoginRequiredMixin, View):
+    """Base view for toggling likes on objects"""
+    model = None
+    template_name = None
+    context_object_name = None
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)